### DISCUSSION TITLE
Challenge Hash on Farming screen is the same for all table entries

### DISCUSSION THREAD

**Discussion by 125devin** [Chia-Network/chia-blockchain#6778] [Support]
I am using Windows 10 with chia 1.1.7, and the Challenge Hash on the Farming screen is the same for all of the table entries in both the Last Attempted Proof table and the Latest Block Challenges table. What does this mean?

**Comment by loppefaaret** [comment]
it is a 10 minute long signage point chain, it is this hash you are seeing - it should change on average every 10 minutes

**Comment by BrandtH22** [comment]
@125devin , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
